plutus-scb-client/src/MainFrame.purs

```diff
@@ -127,6 +122,9 @@ handleAction (ChainAction subaction) = do
   mAnnotatedBlockchain <-
     peruse (_fullReport <<< _Success <<< _chainReport <<< _annotatedBlockchain <<< to AnnotatedBlockchain)
   let
+    wrapper ::
+      Warn (Text "The question, 'Should we animate this?' feels like it belongs in the Chain module. Not here.") =>
```

Comment:
do you want compiler warnings in `master` branch?

Reply:
I want them more than I want `TODO` markers or no reminder at all. But they really shouldn't stay there for long.